OneCommunityGlobal/HighestGoodNetworkApp
# Shihao - Unit Test for TimeZoneDropDown

# Description
Unite test for `TimeZoneDropDown.jsx`

## Main changes explained:
- Add the unit test

## How to test:
1. check into current branch
2. do npm install
3. do `npm test TimeZoneDropDown.test.js` to run the test
4. Verify that all 7 tests have passed

## Screenshots or videos of changes:
<img width="706" alt="Screenshot 2024-01-08 at 14 11 34" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/0296546a-09ca-4d60-bae7-4927641c1a61">
